react/features/overlay/components/AbstractReloadableOverlay.js

```diff
@@ -0,0 +1,20 @@
+/* global APP */
+import AbstractOverlay from './AbstractOverlay';
+
+/**
+ * Implements an abstract React Component for overlay with ability to
+ * reload the page.
+ */
+export default class AbstractReloadableOverlay extends AbstractOverlay {
```

Comment:
Actually I'll move the method to `AbstractOverlay` class.

I started implementing the filmstrip only version and it turned out that I have to create `AbstractReloadableOverlay` and also `AbstractOverlay` for filmstrip only mode also. That way  we have to have +2 more files only because of that method :) So you were right :)